bool[] gripper_closed
int32[] block_in_gripper
int32[] stack
int32[] table
geometry_msgs/Pose[] block_poses